jwst/datamodels/schemas/pathlosscorr.schema.yaml

```diff
@@ -5,30 +5,9 @@ properties:
     fits_hdu: PATHLOSS_POINTSOURCE2D
```

Comment:
I would suggest removing the "2D" suffix, because we no longer need to identify it as such, and perhaps just shorten the whole name (for readability and typability) to just "PATHLOSS_POINT".

Reply:
I would suggest we remove the `_point` and `_uniform` from the attached name.  These 2D corrections are of the same type so they should have the same extension name.  The pipeline processing and SRC_TYPE keyword tells which was done.  This means that anything that needs to use these downstream can just read the `pathloss` extension instead of having to figure out whether it should read the `pathloss_point` or `pathloss_uniform` extension.

Reply:
That's a good point (the potential use downstream needing to know which name to use). I was thinking of keeping points/uniform in the names just as history or logging, but you've convinced me that keeping them generic is probably best.